server/jobs/moss.py

```diff
@@ -72,6 +88,13 @@ def submit_to_moss(moss_id=None, file_regex=".*", assignment_id=None, language=N
             for file in contents:
                 if file == 'submit':  # ignore fake file from ok-client
                     continue
+                if subtract_template and file in assign.files:
+                    # Compare to template and only include lines that new
+                    template, source = assign.files[file], contents[file]
+                    d = difflib.Differ()
```

Comment:
I think we should make difflib ignore spacing and new lines (Extra indentation, Tabs v Spaces ...)  - we can do that with the two args to `Differ` (Junk Char being one of them)